File: test/java/arraysandstrings/ReverseStringTest.java
package java.arraysandstrings;

import org.junit.jupiter.api.Test;

import static java.arraysandstrings.ReverseString.reverse;
import static org.junit.jupiter.api.Assertions.*;

class ReverseStringTest {

    @Test
    void reverseTest() {
        assertEquals("weivretni gnidoC", reverse("Coding interview"));
    }
}

File: src/test/kotlin/arraysandstrings/twopointers/ReverseStringTest.kt
package arraysandstrings.twopointers

import arraysandstrings.tw.ReverseString
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class ReverseStringTest {
    @Test
    fun reverse() {
        Assertions.assertEquals("weivretni gnidoC", ReverseString().reverse("Coding interview"))
    }

}